src/Courses/TopicSettings/HasEverBeenActiveWarning.tsx

```diff
@@ -7,7 +7,7 @@ interface HasEverBeenActiveWarningProps {
 }
 
 export const HasEverBeenActiveWarning: React.FC<HasEverBeenActiveWarningProps> = ({topic}) => {
-    if (topic.isExam() && topic.hasEverBeenActive()) return null;
+    if (!topic.isExam() && topic.hasEverBeenActive()) return null;
```

Comment:
Shouldn't this be
```Typescript
    if (!topic.isExam() || !topic.hasEverBeenActive()) return null;
```
so homework or never active doesn't show the alert
the alert only shows if it is an exam and has been active

Reply:
Ach, good catch.

Reply:
This is why they say to not write negative conditionals.